src/components/cryptoDetails.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import HTMLReactParser from 'html-react-parser';
+import { useParams } from 'react-router-dom';
+import millify from 'millify';
+import {
+  Col, Row, Typography,
+} from 'antd';
+import {
+  MoneyCollectOutlined, DollarCircleOutlined, FundOutlined, ExclamationCircleOutlined, StopOutlined, TrophyOutlined, CheckOutlined, NumberOutlined, ThunderboltOutlined,
+} from '@ant-design/icons';
+
+import { useGetCryptoDetailsQuery } from '../services/cryptoApi';
+import Loader from './Loader';
+
+const { Title, Text } = Typography;
+
+function CryptoDetails() {
+  const { coinId } = useParams();
```

Comment:
- [ ] Please add a back button that links you back to the home page as required. 👌

Reply:
Thank you @Lucas-Masaba , Noted all required changes.